Source/WebDriver/CMakeLists.txt

```diff
@@ -66,13 +66,19 @@ MAKE_JS_FILE_ARRAYS(
 )
 list(APPEND WebDriver_SOURCES ${WebDriver_DERIVED_SOURCES_DIR}/WebDriverAtoms.cpp)
 
-WEBKIT_EXECUTABLE_DECLARE(WebDriver)
+if (ANDROID AND (PORT STREQUAL WPE))
+    set(WebDriver_TARGET_TYPE LIBRARY)
+else ()
+    set(WebDriver_TARGET_TYPE EXECUTABLE)
+endif ()
+
+cmake_language(CALL "WEBKIT_${WebDriver_TARGET_TYPE}_DECLARE" WebDriver)
 
 add_dependencies(WebDriver JavaScriptCoreSharedScripts)
 
 WEBKIT_INCLUDE_CONFIG_FILES_IF_EXISTS()
 
-WEBKIT_EXECUTABLE(WebDriver)
+cmake_language(CALL "WEBKIT_${WebDriver_TARGET_TYPE}" WebDriver)
```

Comment:
Today I learned. `cmake_language()` is simultaneously amazing and terrifying....